### DISCUSSION TITLE
Getting p values from print summary output

### DISCUSSION THREAD

**Discussion by dian-farah** [google/lightweight_mmm#149] [Q&A]
Hi! 

Are we able to get the p values for each of the variable from the model summary output?
![image](https://user-images.githubusercontent.com/66090549/213388102-a02ce4a8-80a3-4cdc-be19-e6e097521ffb.png)
![image](https://user-images.githubusercontent.com/66090549/213388136-e74981ea-2aa7-430f-930e-907741e8b1aa.png)

Thank you!

**Comment by ZenniSherry** [comment]
How to identify the significant level for each media variables and additional features?
